bin/uuid

```diff
@@ -30,6 +32,14 @@ switch (version) {
     var uuidV3 = require('../v3');
 
     var name = args.shift();
+
+    // if two more arguments remain, see if an ecoding has been specified in place of the expected name and process accordingly
+    if (args.length == 2 && /^--encoding=([a-z0-9]+)$/.test(name)){
```

Comment:
The regex is used multiple times. Consider defining it as a variable in the beginning.